themes/custom/apigee_kickstart/webpack.mix.js

```diff
@@ -7,7 +7,7 @@
  | for your application. See https://github.com/JeffreyWay/laravel-mix.
  |
  */
-const proxy = 'http://drupal.local';
+const proxy = 'http://ks.test';
```

Comment:
My guess is this isn't an important value, but what's it used for?

Reply:
@kscheirer This is for [Browsersync](https://www.browsersync.io/).